mosviz/utils/cutout_tools.py

```diff
@@ -34,6 +34,8 @@ def make_cutouts(catalogname, imagename, image_label,
         * ``'dec'`` - DEC (e.g., in degrees).
         * ``'cutout_x_size'`` - Cutout width (e.g., in arcsec).
         * ``'cutout_y_size'`` - Cutout height (e.g., in arcsec).
+        * ``'cutout_pa'`` - Cutout angle (e.g., in degrees). This is only
```

Comment:
No, ideally the tools will handle unit conversion behind the scenes (but I need to clean some things up for that to happen 100%). "e.g." here gives an example of what kind of unit is expected, for clarity. Although realistically speaking, for this sprint, you better give whatever unit that is recommended. :sweat_smile: